openedx/core/djangoapps/api_admin/forms.py

```diff
@@ -0,0 +1,44 @@
+"""Forms for API management."""
+from django import forms
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.api_admin.models import ApiAccessRequest
+
+
+class ApiAccessRequestForm(forms.ModelForm):
+    """Form to request API access."""
+
+    company_name = forms.CharField(help_text=_('The name of your company.'))
```

Comment:
Just following the ACs on that one. @pbaruah may comment further, but the ticket is explicitly written as saving different information than what's taken in on the form.